apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1918,9 +1918,13 @@ pay_close_mutual_fee(Fee, IAmt, RAmt) ->
        true                                   -> {0, RAmt - Fee + IAmt}
     end.
 
+-ifdef(TEST).
 close_mutual_defaults(_Account, _D) ->
-    #{ fee   => 20000}.
-
+    #{ fee   => 20000 }.
+-else.
+close_mutual_defaults(_Account, _D) ->
+    #{ fee   => 20000 * 1000000000 }.
```

Comment:
In this same file I see `default_fee(_Tx) -> 20000.` used in multiple places. I have not got my head around that yet. Hence:
* Not sure if any points using default_fee require amendments.
* I see that this close_mutual_defaults is used as first element in `maps:merge(Def, Opts)` - not sure if any Opts has fee hence overwriting this. (And making this changed line ineffective.)

Reply:
This is [tracked](https://www.pivotaltracker.com/story/show/164072144).